pkg/tools/grpcutils/listen_and_serve.go

```diff
@@ -77,7 +77,7 @@ func ListenAndServe(ctx context.Context, address *url.URL, server *grpc.Server)
 		if _, err = os.Stat(target); os.IsExist(err) {
```

Comment:
no EEXIST error is returned if the file exists (dead code in parenthesis)

Reply:
Indeed we can do something like this 
```
suggestion if _, err = os.Stat(target); err == nil { 
```